plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -190,7 +190,7 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
         );
       }
 
-      return otelContext.with(trace.setSpan(otelContext.active(), span), () => {
+      return otelContext.with(trace.setSpan(parent, span), () => {
```

Comment:
Just curious, why was this `otelContext.active()` before? I'm assuming even the regular AWS Lambda Sampled Context worked because of this code which set the span in the `parent` context. But why do we need this `setSpan` as well as the `startSpan`?

https://github.com/open-telemetry/opentelemetry-js-contrib/blob/47700e10dc6e4bd9ba0255cae85dec07ab4dd448/plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts#L166-L180

Reply:
I assume it was originally missed, because the context was indeed propagated (but the baggage wasn't). We need `startSpan` to make the Lambda span a child of the extracted context. We need the `setSpan` to make the Lambda execution a child of the Lambda span

Reply:
Of course, all the other tests are validating just that (e.g. `takes sampled custom context over sampled lambda context if "eventContextExtractor" is defined`, `uses lambda context if sampled and http context present`, etc.)